src/routes/webhook/webhook.service.ts

```diff
@@ -104,6 +110,16 @@ export class WebhookService {
           return of(undefined);
         }),
       ),
-    );
+    ).then((response: AxiosResponse | undefined) => {
+      if (response) {
+        const endTime = Date.now();
+        const elapsedTime = endTime - startTime;
+        const responseData = this.parseResponseData(response.data);
+        this.logger.debug(
+          `Success sending event ${strMessage} to ${url}: ${response.status} - ${responseData} [Timestamp: ${startTime}, Response time: ${elapsedTime}ms]`,
```

Comment:
Could we use camelcase or similar for _Response time_ to be easier to parse @falvaradorodriguez ?